supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,30 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[command-line-commands-that-require-elevated-privileges]]
+== Command-line commands that require elevated privileges
+
+Some command-line interface (CLI) commands require that a user run them with "elevated privileges".  
+That is, some commands require that a normal user switch to another user account whose privileges permit certain operations or require that a normal user temporarily change their current privileges without changing user accounts.
+
+=== Guidelines for documenting CLI commands with elevated privileges
+
+When documenting CLI commands that require elevated privileges, consider the following guidelines and formulate a consistent approach to use throughout your content:
+
+* If you include a shell prompt in a sample command, always show the correct prompt for a normal user (`$`) or a user with elevated privileges (`#`).
+* Avoid relying solely on a shell prompt in a sample command to indicate the required privilege level to run a command. 
+If a command or series of commands requires elevated privileges, add phrasing to the introductory sentences to indicate that this is the case. For example, "Connect to the cluster node and become the root user".
++
+[NOTE]
+====
+Avoid using variations such as "become root" or similar phrasing.
+====
++
+* If a single command requires temporarily elevated privileges, use `sudo` in the sample command. Do not use `su` without good cause.
```

Comment:
Add an intro statement and then list the two _if_ statements as bullet points?
--------------------------------------
The statement "Do not use `su` without good cause", might be hard to define. How about: Reserve the use of `sudo` for system administrative tasks that cannot be completed with a regular user account.

Reply:
@dfitzmau I think the distinction here is between switching to a user with elevated privileges (su) and entering a single command with elevated privileges (sudo). The guidance is to avoid switching to a user with elevated privileges.

Reply:
Thank you for clarifying! Makes sense. The "normal" user would be another privileges branch?

Reply:
The normal user would be a non-elevated user. In my case, it would be sgoodman. When I need elevated privileges, i.e. to run a command with root privileges, such as `subscription-manager --register`, I need to do it in either of the following ways:
`$ sudo subscription-manager --register`
or
`$ su -i`
`#  subscription-manager --register`

If I'm going to be running a series of commands in which I need to have root (i.e. elevated) privileges, I would rather use `su -i` to switch to a user with root privileges. But if I only need to run one command with elevated privileges, then I'd rather use sudo.

Reply:
Thanks for clarifying, @stoobie . Any need to stipulate the three types of user scenarios somewhere?

Reply:
@dfitzmau Good question. @redhat-documentation/ccs-style-council What do you think?

Reply:
@stoobie  and @dfitzmau - ptal at the current draft version that (I hope) addresses this issue.

Reply:
Thanks for making the updates @bburt-rh . 

Possible to drop the quotation for the example at the end of the content block?

How about:

> For example: The commands in this procedure require elevated privileges to run. Either use `su -i` to switch to the root user or preface the commands with `sudo`.

Reply:
@bburt-rh  Your changes look good to me.